### PR TITLE
Created the note deletion page UI

### LINKED ISSUE #2: Create note deletion page UI

### PR DESCRIPTION
**Pull request by allenjiji**
Fixes #2 
image
image

@yedhink _a 
Please review.
Also corrected the mistakes pointed out in [previous PR](https://github.com/allenjiji/neeto-ui-challenge-allenjiji/pull/7#pullrequestreview-1110745662)

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Card/CardOptionsDropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  setSelectedNoteId,
+}) => {
+  const handleDelete = () => {
+    setShowDeleteAlert(true);
+    setSelectedNoteId([noteId]);
+  };
+  return (
+    <Dropdown buttonStyle="text" icon={MenuVertical}>
+      <li> Edit </li>
+      <li onClick={() => handleDelete()}>Delete</li>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Card/CardOptionsDropdown.jsx:18]: Use shorthand if we don't need to pass in any other args other than the implicit ones:
```suggestion
<li onClick={handleDelete}>Delete</li>
```

File: app/javascript/src/components/Dashboard/SideMenuBar.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // @ts-nocheck
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/SideMenuBar.jsx:1]: https://bigbinary.slack.com/archives/C03F83LDL0L/p1663231277516539

**Review by yedhink — COMMENTED** [review]
@allenjiji

File: app/javascript/src/components/Dashboard/SideMenuBar.jsx (hunk 2 above)
Comment (allenjiji) [reply app/javascript/src/components/Dashboard/SideMenuBar.jsx:1]: I've tried these changes mentioned in this [PR](https://github.com/bigbinary/wheel/pull/892#issue-1374500584). 
But the squiggly lines are still present. Can you please point out anything I've missed?
Anyway now I've removed the //@ts-nocheck from the file.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/SideMenuBar.jsx:1]: @allenjiji Red squiggly lines over what? If it's `neetoui` imports, then it's already addressed at https://github.com/bigbinary/wheel/pull/892#issue-1374500584

**Pull request merged**